reactor-core-2/src/test/java/com/baeldung/reactor/flux/parallelflux/FluxUnitTest.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.reactor.flux.parallelflux;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+import reactor.test.StepVerifier;
+
+import java.time.Duration;
+
+@Slf4j
+public class FluxUnitTest {
+
+    @Disabled("Manual test - takes time due to heavy computation")
```

Comment:
```suggestion
```
Don't do this - we have a naming convention instead - FluxManualTest - that will avoid it being run in CI

https://github.com/eugenp/tutorials/blob/bb77b94f53ce6c274df07b4e3235caa2435023e0/custom-pmd/src/main/java/com/baeldung/pmd/UnitTestNamingConventionRule.java#L16